lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -101,175 +108,42 @@ func (s *subscriberdbServicer) getDigestInfo(
 	req *lte_protos.ListSubscribersRequest,
 	networkID string,
 	apnsByName map[string]*lte_models.ApnConfiguration,
-	apnResourcesByAPN lte_models.ApnResources,
 ) (*lte_protos.SubscribersDigest, bool) {
 	digestProto, noUpdates := req.Digest, false
-	// This functionality is currently placed behind a feature flag
-	if !s.flatDigestEnabled || req.PageToken != "" {
+	// The flat digest functionality is currently placed behind a feature flag
+	if !s.configs.flatDigestEnabled || req.PageToken != "" {
 		return digestProto, noUpdates
```

Comment:
If turned off, should return `"", false` -- and go ahead and make that clear by returning the literals